### PR TITLE
#166925789 Create UI Mocks for the Dashboard

### PR DESCRIPTION
**Pull request by kodek-sleuth**
#### What does this PR do?
Creates Userdashboard templates

#### Description of Task to be completed
- Create userdashboard
- Create templates for `CRUD`
#### How should this be manually tested?
1. Run the following commands in the terminal
```
git clone https://github.com/andela/ah-92explorers-backend.git
cd ah-92explorers-backend
git checkout ch-userdashboard-template-166925789
```
2. Open your preferred Text-Editor 
3. Run the `html file` in your browser
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[166925789](https://www.pivotaltracker.com/story/show/166925789)
#### Screenshots (if appropriate)
Screen Shot 2019-07-03 at 09 34 24
Screen Shot 2019-07-03 at 09 34 35
Screen Shot 2019-07-03 at 09 34 39

#### Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

**Comment by kipropbrian** [comment]
Hey could you tell us what the svg icons are for?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (Hervera) [inline .gitignore]: Add a new line at the bottom

**Review by Hervera — CHANGES_REQUESTED** [review]
Add a new line at the bottom of your pages

File: .gitignore (hunk 1 above)
Comment (kodek-sleuth) [reply .gitignore]: Man that's markup and css, there is no need for a newline at the bottom. Read the docs well

File: userDashboard.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                <img>
+            </div>
+        </div>
+    </div>
+    
+
+
+    
+</body>
+</html>
```
Comment (Hervera) [inline userDashboard.html:114]: Add a new line

**Review by elemanhillary-zz — APPROVED** [review]
mock ups are ready to be merged

**Review by IsaiahRn — APPROVED** [review]
It looks nice, it can be merged!

File: icons/edit.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...532 16.5625-13.4375 29.984375-30 30h-280.589844c-16.5625-.015625-29.980469-13.4375-30-30v-260.589844c.019531-16.558594 13.4375-29.980469 30-30h88.789062c5.523438 0 10-4.476563 10-10 0-5.519531-4.476562-10-10-10h-88.789062c-27.601562.03125-49.96875 22.398437-50 50v260.59375c.03125 27.601563 22.398438 49.96875 50 50h280.589844c27.601562-.03125 49.96875-22.398437 50-50v-88.792969c0-5.523437-4.476563-10-10-10zm0 0"/><path d="m376.628906 13.441406c-17.574218-17.574218-46.066406-17.574218-63.640625 0l-178.40625 178.40625c-1.222656 1.222656-2.105469 2.738282-2.566406 4.402344l-23.460937 84.699219c-.964844 3.472656.015624 7.191406 2.5625 9.742187 2.550781 2.546875 6.269531 3.527344 9.742187 2.566406l84.699219-23.464843c1.664062-.460938 3.179687-1.34375 4.402344-2.566407l178.402343-178.410156c17.546875-17.585937 17.546875-46.054687 0-63.640625zm-220.257812 184.90625 146.011718-146.015625 47.089844 47.089844-146.015625 146.015625zm-9.40625 18.875 37.621094 37.625-52.039063 14.417969zm227.257812-142.546875-10.605468 10.605469-47.09375-47.09375 10.609374-10.605469c9.761719-9.761719 25.589844-9.761719 35.351563 0l11.738281 11.734375c9.746094 9.773438 9.746094 25.589844 0 35.359375zm0 0"/></svg>
```
Comment (Hervera) [inline icons/edit.svg:1]: Add a new line

**Review by niyoceles — APPROVED** [review]

**Review by Hervera — APPROVED** [review]

**Pull request merged**